hasher-matcher-actioner/hmalib/lambdas/api/collabs.py

```diff
@@ -78,4 +86,53 @@ def get_collab_configs():
         configs = tx_collab_config.get_all_collab_configs()
         return AllCollabsEnvelope(configs)
 
+    @collabs_api.get("/get-schema-for-class")
+    def get_schema_for_class():
+        """
+        Get the schem for a specific class. This could be a new class or one
+        that we already have a config for. Used on the UI to provide a form to
+        create a new config.
+        """
+        try:
+            pytx_collab_config_class = import_class(bottle.request.query["class"])
+        except ModuleNotFoundError:
+            bottle.abort(404)
+
+        return {
+            "class": full_class_name(pytx_collab_config_class),
+            "schema": _to_schema(pytx_collab_config_class),
+        }
+
+    @collabs_api.post("/add-collab-config")
+    def add_collab_config():
```

Comment:
Is there an edit, or only an add?

Reply:
So far, only an add. Even the UI does not allow one. This is because the behaviour when you change something like the privacy group id is undefined. 

Unless we add a bunch of validations, enable / disable would be the only change supported. That can / should be done via the bank.